core/src/main/scala/kafka/admin/AclCommand.scala

```diff
@@ -640,20 +472,8 @@ object AclCommand extends Logging {
       if (options.has(bootstrapServerOpt) && options.has(bootstrapControllerOpt))
         CommandLineUtils.printUsageAndExit(parser, "Only one of --bootstrap-server or --bootstrap-controller must be specified")
 
-      val hasServerOrController = options.has(bootstrapServerOpt) || options.has(bootstrapControllerOpt)
-      if (hasServerOrController && options.has(authorizerOpt))
-        CommandLineUtils.printUsageAndExit(parser, "The --authorizer option can only be used without --bootstrap-server or --bootstrap-controller")
-
-      if (!hasServerOrController) {
-        CommandLineUtils.checkRequiredArgs(parser, options, authorizerPropertiesOpt)
-        System.err.println(AclCommand.AuthorizerDeprecationMessage)
-      }
-
-      if (options.has(commandConfigOpt) && (!hasServerOrController))
-        CommandLineUtils.printUsageAndExit(parser, "The --command-config option can only be used with --bootstrap-server or --bootstrap-controller option")
-
-      if (options.has(authorizerPropertiesOpt) && hasServerOrController)
-        CommandLineUtils.printUsageAndExit(parser, "The --authorizer-properties option can only be used with --authorizer option")
+      if (!options.has(bootstrapServerOpt) && !options.has(bootstrapControllerOpt))
```

Comment:
Can we add a test for this?

Reply:
I think `AclCommandTest#testUseWithoutBootstrapServerOptAndBootstrapControllerOpt` can cover this. Thank you.